src/extensions/extension-discovery.ts

```diff
@@ -387,24 +387,19 @@ export class ExtensionDiscovery extends Singleton {
         await this.installPackage(extension.absolutePath);
       }
     }
-    const extensions = bundledExtensions.concat(userExtensions);
 
-    return this.extensions = new Map(extensions.map(extension => [extension.id, extension]));
+    return this.extensions = new Map(bundledExtensions.concat(userExtensions).map(extension => [extension.id, extension]));
```

Comment:
These changes are out-of-scope for this PR. Could you revert unless they are really needed.